lib/modules/manager/terraform/util.ts

```diff
@@ -18,6 +18,9 @@ export function checkIfStringIsPath(path: string): boolean {
   return !!match;
 }
 
+const ociRegex = regEx(/^oci:\/\//);
+export { ociRegex };
+
```

Comment:
```suggestion
```
Not needed anymore

Reply:
I still need it in the helm extractor, but only there, so moving it over in  `a1af38809b1fc68ce09796626ce964a4708d5252` .

Reply:
I was debating that too, went for the regex because I can match the beginning of the string more easily.